schemas/configuration/v5/server_config.json.in

```diff
@@ -103,7 +103,11 @@
             "type": "integer",
             "minimum": 0
         },
-        "host": {"type": "string"},
+        "host": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 256
```

Comment:
Change to 255.

Reply:
Add a test proving the length requirement is enforced.

Reply:
Done.

Changed maxLength to 253 since that is the true usable character limit.

Reply:
oh?!  why?   if anything, i'd have expected 254 plus null b/c the spec might say 255...

Reply:
ah, behold.   learned something (again).

https://web.archive.org/web/20190518124533/https://devblogs.microsoft.com/oldnewthing/?p=7873